practica2023/src/main/java/com/endava/practica2023/Domain/dto/TicketCategoryDTO.java

```diff
@@ -0,0 +1,9 @@
+package com.endava.practica2023.Domain.dto;
+
+import jakarta.persistence.Column;
+
+public record TicketCategoryDTO(
+    Integer ticket_category_id,
+    String ticket_category_description,
+    Float ticket_category_price
```

Comment:
Naming convention in Java is camel case for methods and variables, not how you name them with _ between words